### ISSUE TITLE
JellyfUn 0.0.3 playback distorted when screen is black

### ISSUE THREAD

**Issue by ghost** [Tewelus/jellyfUn#1]
With JellyfUn configured via UT Tweak Tool to prevent it from stop playing when JellyfUn  goes to the background, playing continues but very much distorted when the screen goes black. As soon as the screen comes back again, even before unlocking, playback is fine again.

**Comment by Tewelus** [comment]
Hello, thank you for your report. Would you please tell on what device and what release-channel your on? I can't reproduce it. Works well on nexus 5, no distortion on black screen.

**Comment by ghost** [comment]
I'm using a OnePlus 3 and release channel is stable. Is there any way I can get some debugging logs for you from my device? Anything else I can help you with, let me know.

**Comment by Tewelus** [comment]
I'd like to debug something, I don't know exactly what. If you feel like it, you can test if there are different results with different songs. There are indications that the sample rate used to cause distortion. Does the problem also occur when you use the browser or run audio through another app in the background with a black screen? Does the problem also occur with the developer channel?

**Comment by ghost** [comment]
Hi, Yes, I changed it to the develop channel, and yes the problem exists also there. Using the browser (Morph) the music just stops when the screen goes black. Most of the song are flac songs. I tried some with bitrates of 722 kbps, 850kbps, 876 kbps and 954 kbps, but I also tried some mp3s with bitrates as low as 239kbps and 213kbps.

 The "distortion" is like this: the song just continues after I turn off the screen for a second or two, and then stops, waits, goes on for a bit, then stops again, etc.

**Comment by Tewelus** [comment]
Just to be sure, if you use the browser to play jellyfin-audio and use UT-Tweak-Tool to prevent the browser from suspension, does it stops playing?

**Comment by ghost** [comment]
No, the problem doesn't appear when Morph plays the song and the screen goes black, when I use the UT Tweak Tool to prevent the browser from suspending, as you suggested.
